docs/reference/docs/reindex.asciidoc

```diff
@@ -52,13 +56,26 @@ That will return something like this:
 --------------------------------------------------
 // TESTRESPONSE[s/"took" : 147/"took" : "$body.took"/]
 
+////
+
+[[docs-reindex-api-request]]
+==== {api-request-title}
+
+`POST /_reindex`
+
+[[docs-reindex-api-desc]]
+==== {api-description-title}
+
```

Comment:
Most of the snippets and subsections in the description could be moved to examples IMO.